chainer/functions/array/get_item.py

```diff
@@ -68,12 +68,13 @@ def get_item(x, slices):
     """Extract elements from array with specified shape, axes and offsets.
 
     Args:
-        x (~chainer.Variable): A variable to be sliced.
+        x (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`): A variable to be sliced.
         slices (int, slice, Ellipsis, None, integer array-like, boolean\
         array-like or tuple of them):
             It is an integer, a slice, an ellipsis,
-            a numpy.newaxis, an integer array-like, a boolean array-like
-            or tuple of them.
+            a numpy.newaxis, a None, an integer array-like,
+            a boolean array-like or tuple of them.
```

Comment:
I think this whole sentence is redundant. How about more general expression, such as "an object to specify the selection of elements"?

Reply:
Sounds much better, thank you!